presto-docs/src/main/sphinx/sql/show-session.rst

```diff
@@ -7,12 +7,13 @@ Synopsis
 
 .. code-block:: none
 
-    SHOW SESSION
+    SHOW SESSION [ LIKE pattern ]
```

Comment:
ESCAPE?

Reply:
We can add but won't it make it a bit complex to read. And for other `SHOW` queries we didn't specify the escape pattern so didn't add it here